sdk/Lusid.Sdk.Tests/tutorials/Ibor/Orders.cs

```diff
@@ -356,5 +357,117 @@ public void Upsert_And_Retrieve_Simple_Orders()
             */
 
         } 
+        [LusidFeature("F8")]
```

Comment:
the features map to the values here https://gitlab.finbourne.com/salesengineering/lusid-features/-/blob/master/features.yml (soon to be made public). `F-8` is already taken! Could you refactor the order ones to be F4-1, F4-2,... (see the tools ones for examples).

The `features` are use to identify which ones have been implemented across the different language sdks